build.gradle

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    // Define versions in a single place
+    ext {
+        // Sdk and tools
+        compileSdkVersion = 30
+        minSdkVersion = 28
+        targetSdkVersion = 30
+
+        // App dependencies
+        appCompatVersion = '1.1.0'
+        assistedInjectVersion = '0.5.2'
+        constraintLayoutVersion = '2.0.0-beta3'
+        coreTestingVersion = '2.0.0'
+        coroutinesVersion = "1.4.2"
+        espressoVersion = '3.1.1'
+        fragmentVersion = '1.1.0-alpha09'
+        glideVersion = '4.10.0'
+        gradleVersion = '4.1.1'
+        gsonVersion = '2.8.2'
+        hiltVersion = '2.28.3-alpha'
+        hiltViewModelVersion = '1.0.0-alpha02'
+        junitVersion = '4.12'
+        kotlinVersion = '1.4.21'
+        ktlintVersion = '0.38.1'
+        ktxVersion = '1.0.2'
+        lifecycleVersion = '2.2.0'
+        materialVersion = '1.2.0'
+        navigationVersion = '2.2.0'
+        okhttpLoggingVersion = '4.7.2'
+        pagingVersion = '3.0.0-alpha02'
+        recyclerViewVersion = '1.2.0-alpha04'
+        retrofitVersion = '2.9.0'
+        roomVersion = '2.2.5'
+        workVersion = '2.5.0'
+        runnerVersion = '1.0.1'
+        truthVersion = '0.42'
+        testExtJunit = '1.1.0'
+        uiAutomatorVersion = '2.2.0'
+        viewPagerVersion = '1.0.0'
+        workVersion = '2.1.0'
+    }
+
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath "com.android.tools.build:gradle:$gradleVersion"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navigationVersion"
+        classpath "com.google.dagger:hilt-android-gradle-plugin:$hiltVersion"
+    }
+}
+
+plugins {
+    id "com.diffplug.spotless" version "5.2.0"
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+spotless {
+    kotlin {
+        target "**/*.kt"
+        ktlint(ktlintVersion).userData(['max_line_length' : '100'])
```

Comment:
great that spotless and ktlint are already setup! it looks like there are a few code style issues that could be fixed by running `spotlessApply`